### PR TITLE
Use std::string::empty instead comparing with an empty string

### PR DESCRIPTION
**Pull request by dawagner**
Caught by clang-tidy.

### CODE REVIEW

File: include/class_loader/class_loader_register_macro.h [hunk 1]
```diff
@@ -46,7 +46,7 @@
     typedef  Base _base; \
     ProxyExec ## UniqueID() \
     { \
-      if (std::string(Message) != "") { \
+      if (not std::string(Message).empty()) { \
```
Comment (mikaelarguedas) [inline include/class_loader/class_loader_register_macro.h]: Nitpick: As the entire library uses `!` for the negation operator and not the alternative operator `not`, I'd prefer to use `!` here

**Review by mikaelarguedas — COMMENTED** [review]
thanks @dawagner for the fix. I have one tiny comment otherwise looks good to me

File: include/class_loader/class_loader_register_macro.h (hunk 1 above)
Comment (dawagner) [reply include/class_loader/class_loader_register_macro.h]: Ok, done.

**Review by mikaelarguedas — APPROVED** [review]
lgtm, thanks!

**Pull request merged**